### PR TITLE
Port to ROS2, part 1: build

### PR DESCRIPTION
**Pull request by nnmm**
Ported the build to use `ament_python` instead of `ament_cmake`, which I think is more idiomatic for Python code in ROS2

## Changes
* Removed the `scripts` subdirectory, since that is the natural package layout for setuptools. `pkg create` does the same.
* Removed the `scenario_loader.py.cmake` file. The only thing in it is a variable with the location of the source files, used to interpret relative paths as relative to the source file location. I don't know how to implement that in setuptools and it's probably not the best idea to do it this way anyway.
* The imports needed to be made package-relative.

## Reviewer comments
* This is only part 1 of porting this package. `COLCON_IGNORE` is still there, but I have a branch where I checked that this works.
* The list of dependencies in package.xml is probably inaccurate, we should revisit this later
* There is also a (incomplete) `requirements.txt`. I'm new to setuptools – should setuptools somehow know about the external dependencies?

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -9,8 +10,9 @@
   <maintainer email="httperror@404-notfound.jp">Tatsuya Yamasaki</maintainer>
 
   <license>Apache 2</license>
```
Comment (esteve) [inline package.xml:12]: Apache License 2.0

File: package.xml [hunk 2]
```diff
@@ -1,5 +1,6 @@
-<?xml version="1.0" ?>
-<package format="2">
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>planning_simulator_launcher</name>
   <version>0.0.0</version>
```
Comment (esteve) [inline package.xml]: IIRC 0.0.0 is not a valid version number for ROS packages, could you bump it to 0.0.1? Thanks.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+from setuptools import setup
+from glob import glob
+
+package_name = 'planning_simulator_launcher'
+
+setup(
+    name=package_name,
+    version='0.0.0',
```
Comment (esteve) [inline setup.py]: 0.0.1 here as well

**Review by esteve — CHANGES_REQUESTED** [review]

**Comment by nnmm** [comment]
Thanks, will do! So I shouldn't worry about requirements.txt?

**Comment by esteve** [comment]
@nnmm `requirements.txt` should be updated to reflect the dependencies, but I don't know which ones should go in there. `pip freeze` will give you a list of all the Python packages, but you can't know for sure which dependencies are actually used for this package.

**Review by esteve — APPROVED** [review]

**Pull request merged**